### ISSUE TITLE
Summaries can contain wrongly markdown'd content

### ISSUE THREAD

**Issue by jpluscplusm** [cloudhead/toto#33]
A summary that has pertinent markdown syntax beyond :max (or :delim, but that's more the author's fault) can display wrongly formatted content on the index page.

The example I've found is the <code> backtick (currently visible on http://jpluscplusm.com/blog/).  If the opening backtick is before :max and the closing backtick is after :max, then

a) the backtick is included in the output and
b) it is ignored for formatting purposes.

Obviously the fix is to manually include a :delim character before :max in an appropriate place, but there might be something clever that could be done to make this a non-issue for articles without a :delim.

Something that came to mind was reversing the logic in def(summary) from markdown(truncated_body) to truncate(markdown(entire_body)).  Unfortunately, I don't think this would work very well, probably resulting in a syntactically invalid summary when it's truncated halfway through an HTML tag.

**Comment by nogweii** [comment]
Clone of #18 - the same fix for that applies to this one as well.

The "clever" hackery I came up with is to use Nokogiri on the parsed, complete HTML version of the Markdown. See evaryont/mindtables@0054653bfbba57589ee01fa762edfa37c10eb5d8 (with a fix at evaryont/mindtables@70b5c27666f734f8c6bf76839e9c6736b52243f9 ) for how I did it. This isn't the best way, as the results aren't cached, but it works, pretty much.
